R/Learner.R

```diff
@@ -566,21 +562,11 @@ Learner = R6Class("Learner",
       assert_character(rhs)
       assert_names(names(rhs), subset.of = c("train", "predict"))
       private$.encapsulate = insert_named(default, rhs)
-
-      # if there is no fallback, we get a default one
-      if (is.null(private$.fallback)) {
-        fallback = default_fallback(self)
-        if (!is.null(fallback)) {
-          self$fallback = fallback
-        }
-      }
     },
 
     #' @field fallback ([Learner])\cr
```

Comment:
i would check a little bit more in terms of properties.

if these properties are available in the learner, but NOT the FB I would warn now
2class, multiclass, missings, selected features, 
predict_type the same should be an extra message